app/models/work_completion/plate_completion.rb

```diff
@@ -8,21 +8,28 @@
 # @author Genome Research Ltd.
 #
 class WorkCompletion::PlateCompletion
-  attr_reader :target_plate, :submission_ids
+  attr_reader :target_plate, :submission_ids, :order_ids
```

Comment:
I know this is an existing issue, but if these two classes (PlateCompletion and TubeCompletion) share attributes and the logic is quite similar, should they have a shared super class?  Or will that not work because of Rails?

Reply:
Thanks, I think it would make sense. To be honest I kept reading them as if they inherited from WorkCompletion - so had similar thoughts.